features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/APIStateChange.jsx

```diff
@@ -123,10 +129,13 @@ function ListLabels() {
     }, []);
 
     const updateStatus = (referenceId, value) => {
-        const body = {};
-        body.status = value;
-        body.attributes = {};
-        body.description = 'Approve workflow request.';
+        const body = { status: value, attributes: {}, description: '' };
+        if (value === 'APPROVED') {
+            body.description = 'Approve workflow request.';
```

Comment:
Internationalise these texts

Reply:
the description is a value sent to the database. So didn't update as a static message.